renting/setting-up-renterd/linux.md

```diff
@@ -224,124 +281,185 @@ User=renterd
 WantedBy=multi-user.target
 Alias=renterd.service
 ```
+{% endtab %}
+
+{% tab title="Zen Test Net" %}
+```toml
+[Unit]
+Description=Sia renterd (Zen Testnet)
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/renterd_zen
+WorkingDirectory=/var/lib/renterd_zen
+Restart=always
+RestartSec=15
+User=renterd_zen
+
+[Install]
+WantedBy=multi-user.target
+Alias=renterd_zen.service
+```
+{% endtab %}
+{% endtabs %}
 
 You can now save the file with `ctrl+s` and exit with `ctrl+x`.
 
+---
+
 ## Running `renterd`
 
 Now it is time to start the service
 
+{% tabs %}
+{% tab title="Main Net" %}
 ```console
 sudo systemctl start renterd
 ```
+{% endtab %}
 
-Your `renterd` service should now be running. You can check the status of the service by running the following command:
+{% tab title="Zen Test Net" %}
+```console
+sudo systemctl start renterd_zen
+```
+{% endtab %}
+{% endtabs %}
 
+Your `renterd` service should now be running. You can check the status of the service by running the following command:
+  
+{% tabs %}
+{% tab title="Main Net" %}
 ```console
 sudo systemctl status renterd
 ```
+{% endtab %}
 
-If the service was set up correctly, it should say "active (running)."
-![Starting renterd](../../.gitbook/assets/renterd-success.png)
+{% tab title="Zen Test Net" %}
+```console
+sudo systemctl status renterd_zen
+```
+{% endtab %}
+{% endtabs %}
 
-3. You can now access the `renterd` web UI by opening a browser and going to `http://localhost:9980`.
+If the service was set up correctly, it should say “active (running).”
```

Comment:
Include checking `journalctl -fu renterd` to see console output in case it
doesn't start